packages/docs/docs/troubleshooting.md

```diff
@@ -80,3 +80,7 @@ Here is what you can try when the extension got stuck on some errors:
 If you need to install an older version of an IDE, you can do so by navigating to the cogwheel menu next to the "install" button in the market place. 
 
 <img width="698" alt="download-older-version" src="/img/docs/marketplace_install_older_version.png"/>
+
+### -sec-num- Configureing Alternative Xcode Versions
+
+If you are using alternative Xcode version ( e.g. xcode-beta, "xcodes" ) IDE will only work if xcode-select points to the correct directory to set it up run: `xcode-select --switch ${PathToYourXCode}/Contents/Developer`
```

Comment:
can we add link to xcodes here such that it is clear for people to understand what we refer to here in case they don't know what they use.

Also: have you tried xcodes here? It seems like it has `select` command so I'd expect that it sets the value with xcode-select, otherwise building swift/xcode projects from command line wouldn't work? Regardless of that I think we can keep this in the instruction, but it'd be good to know whether we should expect this to work when someone is using xcodes or not